src/main/java/uk/gov/pay/api/model/ValidCreatePaymentRequest.java

```diff
@@ -34,7 +34,7 @@
     private Boolean delayedCapture;
 
     public ValidCreatePaymentRequest(CreatePaymentRequest createPaymentRequest) {
-        amount = Objects.requireNonNull(createPaymentRequest.getAmount());
+        amount = createPaymentRequest.getAmount();
```

Comment:
This change confused me at first but then I realised that `createPaymentRequest.getAmount()` returns an `int`, which cannot be null

Reply:
As per the corresponding commit message ;)